UACloudLibraryServer/GraphQL/UaCloudLibResolver.cs

```diff
@@ -196,109 +199,109 @@ private List<long> ApplyWhereExpression(string where)
             return nodesetIds;
         }
 
-        public Task<List<AddressSpace>> GetAdressSpaceTypes(int limit, int offset, string where, string orderBy)
+        public Task<List<UANameSpace>> GetAdressSpaceTypes(int limit, int offset, string where, string orderBy)
```

Comment:
Missed an "AddressSpace".

Reply:
Misspelt addressspace got me, thanks for finding this! :-)